lib/ansible/module_utils/vmware.py

```diff
@@ -623,3 +623,18 @@ def serialize_spec(clonespec):
             data[x] = str(xt)
 
     return data
+
+
+def find_host_by_cluster_datacenter(module, content, datacenter_name, cluster_name, host_name):
+    dc = find_datacenter_by_name(content, datacenter_name)
+    if dc is None:
+        module.fail_json(msg="Unable to find datacenter with name %s" % datacenter_name)
+    cluster = find_cluster_by_name(content, cluster_name)
```

Comment:
Shouldn't this be `find_cluster_by_name(content, cluster_name, dc)`? Otherwise, getting the datacenter is a bit useless